### PR TITLE
Added comment in the ModeCompleted.msg

### PR DESCRIPTION
**Pull request by federicociresola**
I added a little comment where a person can see the possible value of nav_state

### CODE REVIEW

File: msg/ModeCompleted.msg [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Mode completion result, published by an active mode.
+# Mode completion result, published by an active mode. The possible values of nav_state are defined 
+# in the [source VehicleStatus](https://github.com/PX4/PX4-Autopilot/blob/main/msg/VehicleStatus.msg).
```
Comment (hamishwillee) [inline msg/ModeCompleted.msg:2]: I think first sentence gets used for the one line description, so slightly ordering.
```suggestion
# Mode completion result, published by an active mode.
# The possible values of nav_state are defined in the 
# [source VehicleStatus](https://github.com/PX4/PX4-Autopilot/blob/main/msg/VehicleStatus.msg).
```

File: msg/ModeCompleted.msg [hunk 2]
```diff
@@ -10,5 +11,5 @@ uint8 RESULT_FAILURE_OTHER = 100 # Mode failed (generic error)
 
 uint8 result                     # One of RESULT_*
 
-uint8 nav_state                  # Source mode
+uint8 nav_state                  # Source mode (see it in VehicleStatus)
```
Comment (hamishwillee) [inline msg/ModeCompleted.msg:14]:
```suggestion
uint8 nav_state                  # Source mode (values in VehicleStatus)
```

File: msg/ModeCompleted.msg [hunk 3]
```diff
@@ -1,4 +1,6 @@
 # Mode completion result, published by an active mode.
+# The possible values of nav_state are defined in the 
+# [VehicleStatus](https://github.com/PX4/PX4-Autopilot/blob/main/msg/VehicleStatus.msg) msg.
```
Comment (hamishwillee) [inline msg/ModeCompleted.msg:3]:
```suggestion
```

File: msg/ModeCompleted.msg [hunk 4]
```diff
@@ -10,5 +12,5 @@ uint8 RESULT_FAILURE_OTHER = 100 # Mode failed (generic error)
 
 uint8 result                     # One of RESULT_*
 
-uint8 nav_state                  # Source mode
+uint8 nav_state                  # Source mode (values in VehicleStatus)
```
Comment (hamishwillee) [inline msg/ModeCompleted.msg:15]:
```suggestion
uint8 nav_state                  # Source mode (values defined in VehicleStatus msg)
```

**Review by hamishwillee — APPROVED** [review]
@bkueng I'm happy with this, but would also have been happy with just the comment on line 15. Does it make sense to you too?

----

As an aside, it would help me to discuss at some point what docs are useful generally for topics and update them all at the same time.
It might even be worth adding some tagging for this - `@short`, `@description`, `@instances` (something to indicate whether there can be multiples). Also to decide whether we allow markdown in the comments and so on. 

For example, if we allowed something like:

```
uint8 nav_state                  # Source mode (`VehicleStatus`)
```

Then I could know VehicleStatus is a message and auto-cross link or similar.
Just a thought.

File: msg/ModeCompleted.msg (hunk 3 above)
Comment (hamishwillee) [inline msg/ModeCompleted.msg:3]: I can autolink later.

```suggestion
# The possible values of nav_state are defined in the VehicleStatus msg.
```

**Review by bkueng — APPROVED** [review]
> It might even be worth adding some tagging for this - @short, @description, @instances (something to indicate whether there can be multiples). Also to decide whether we allow markdown in the comments and so on.

I'm all for extending the metadata & docs around that. Ideally we do this in ROS and I started a discussion a while ago, but it did not make it any further.

**Pull request merged**